src/desktop/apps/armory_week/fixture.ts

```diff
@@ -0,0 +1,103 @@
+export default {
```

Comment:
had to change the fixture from json to ts, for some reason wasn't working, was an easy-go solution